ion/src/shared/timing.cpp

```diff
@@ -0,0 +1,9 @@
+#include <ion.h>
+#include <chrono>
+
+static auto start = std::chrono::high_resolution_clock::now();
+
+uint32_t Ion::millis() {
+  auto elapsed = std::chrono::high_resolution_clock::now() - start;
```

Comment:
We should switch from `std::chrono::high_resolution_clock` to `std::chrono::steady_clock` to match the device's behavior.

Reply:
Good idea, thanks!